calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -146,16 +140,15 @@ Before you get started, make sure you have downloaded and configured the [necess
     helm repo update
     ```
 
-1. Install version {{releaseTitle}} of the {{prodname}} operator and custom resource definitions.
-
+1. Create the tigera-operator namespace.
     ```batch
-    helm install calico projectcalico/tigera-operator --version {{releaseTitle}}
+    kubectl create namespace tigera-operator
     ```
 
-1. Finally, add nodes to the cluster.
+1. Install version {{releaseTitle}} of the {{prodname}} operator and custom resource definitions.
```

Comment:
`releaseTitle` returns the current version of Calico. If you're looking for the current version of the operator, you'll need `tigeraOperatorVersionShort` or `tigeraOperator`. See variables.js and releases.json in the product/version root directory.

Reply:
the {{prodname}} operator

Isn't it still called the Tigera operator?

Reply:
```suggestion
1. Install version {{releaseTitle}} of the tigera-operator and custom resource definitions.
```